src/configuration.ts

```diff
@@ -66,7 +66,7 @@ export function appDir(): string {
 		(process.platform === "win32"
 			? path.resolve(process.env.LOCALAPPDATA!, packageDotJson.name)
 			: path.resolve(process.env.HOME!, `.${packageDotJson.name}`));
-	accessSync(appDir);
+	accessSync(appDir, constants.R_OK | constants.W_OK);
```

Comment:
I forgot to add the access mode in my "errors on invalid config" PR - this access checks for when the Docker daemon creates the config dir on a user's behalf, owned by root so cross-seed can't do anything if it's running as a non-root user.